tensorboard/plugins/core/index.html

```diff
@@ -1,2 +1,19 @@
+<!--
+@license
+Copyright 2018 The TensorFlow Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
 <!doctype html>
```

Comment:
This breaks CorePluginTest - https://github.com/tensorflow/tensorboard/pull/1547 should fix that by moving this into a memory-only zipfile for faking out the assets_zip_provider.